### DISCUSSION TITLE
Mods doesn't work

### DISCUSSION THREAD

**Discussion by Sbst397** [thatskymod/Sky-CotL-Scripts#115] [Q&A] [answered]
I have a little problem when I use the music mod it immediately disconnected my account and it doesn't work at all, and it's the same for dukun langit, it doesn't work too, if the creator of those mods can fix it would be wonderful and if someone know why it doesn't work anymore I would like to know I did everything to make it work but it doesn't

**Comment by thatskymod** [comment, marked as answer]
Mods in this repository are regularly tested for compatibility. That said, I don't see any apparent issue with the current mods. Be sure to update libLangit to the latest version, [0.4.5](https://github.com/thatskymod/Sky-CotL-Scripts/blob/6a5d6946eb5661e5e14f714d56ca0f4157b3c351/Canvas/Libs/libLangit045.so). As for libsMusic, it no longer receives support or feature updates from its author, so you should remove it to prevent Canvas from crashing. Same applies to your other thread, #117.
